dll/win32/shell32/folders/CFSFolder.cpp

```diff
@@ -1499,47 +1499,41 @@ HRESULT WINAPI CFSFolder::SetNameOf(
     }
 
     /* build source path */
-    PathCombineW(szSrc, m_sPathTarget, pDataW->wszName);
+    PathCombineW(szSrc, m_sPathTarget, pDataW->wszName); // FIXME: PIDLs without wide string
```

Comment:
What do you mean? Do you mean that what's stored in wszName is not a wide string? or?

Reply:
No, I mean there are pidls with only the ASCII name. Windows still generates them today.

Calling _ILSimpleGetTextW would be less broken but it is still wrong. The correct solution is for this file to interpret its own pidls.

Reply:
Ah, so I understand it's still an unimplemented thing.